server/models/Transaction.js

```diff
@@ -80,7 +99,12 @@ export default (Sequelize, DataTypes) => {
           paymentProcessorFee: this.paymentProcessorFee,
           netAmountInGroupCurrency: this.netAmountInGroupCurrency,
           amountInTxnCurrency: this.amountInTxnCurrency,
-          txnCurrency: this.txnCurrency
+          txnCurrency: this.txnCurrency,
+
+          // expense related fields
+          expenseCategory: this.expenseCategory,
+          expenseIncurredAt: this.expenseIncurredAt,
+          expensePayoutMethod: this.expensePayoutMethod
```

Comment:
Same reason we don't send back the entire transaction object but send .public. You don't want to send entire expense object. It's unnecessary and creates parsing work on front end. 

Think of those fields as Transaction fields but happen to be on Expense table. They should be included with every Transaction request sent back. 

By including them here, we don't have to worry about populating them on every controller that returns Transaction. 

Open to other solutions.